prac_07/programming_language.py

```diff
@@ -7,16 +7,18 @@
 class ProgrammingLanguage:
     """Represent information about a programming language."""
 
-    def __init__(self, name, typing, reflection, year):
+    def __init__(self, name, typing, reflection, pointer_arithmetic, year):
```

Comment:
Consistent Indentation: Ensured that the code follows a consistent four-space indentation as per PEP 8 guidelines​.